.github/workflows/codespell_and_flake.yml

```diff
@@ -16,19 +16,30 @@ jobs:
       CODESPELL_SKIPS: 'doc/auto_*,*.fif,*.eve,*.gz,*.tgz,*.zip,*.mat,*.stc,*.label,*.w,*.bz2,*.annot,*.sulc,*.log,*.local-copy,*.orig_avg,*.inflated_avg,*.gii,*.pyc,*.doctree,*.pickle,*.inv,*.png,*.edf,*.touch,*.thickness,*.nofix,*.volume,*.defect_borders,*.mgh,lh.*,rh.*,COR-*,FreeSurferColorLUT.txt,*.examples,.xdebug_mris_calc,bad.segments,BadChannels,*.hist,empty_file,*.orig,*.js,*.map,*.ipynb,searchindex.dat,install_mne_c.rst,plot_*.rst,*.rst.txt,c_EULA.rst*,*.html,gdf_encodes.txt,*.svg'
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: GuillaumeFavelier/ci-skip@enh/separator
+        with:
+          commit-filter: '[skip ci];[ci skip];[skip github]'
+          separator: ';'
       - uses: actions/setup-python@v2
+        if: ${{ env.CI_SKIP == 'false' }}
         with:
           python-version: '3.8'
           architecture: 'x64'
       - run: |
           python -m pip install --upgrade pip setuptools wheel
           python -m pip install flake8
         name: 'Install dependencies'
+        if: ${{ env.CI_SKIP == 'false' }}
       - uses: rbialon/flake8-annotations@v1
+        if: ${{ env.CI_SKIP == 'false' }}
         name: 'Setup flake8 annotations'
       - run: make flake
+        if: ${{ env.CI_SKIP == 'false' }}
         name: 'Run flake8'
       - uses: GuillaumeFavelier/actions-codespell@feat/quiet_level
+        if: ${{ env.CI_SKIP == 'false' }}
```

Comment:
Do we have to do it this way? No way to say "don't do anything after this step" and put this once at the top?

Reply:
This is the way I know and it's also consistent with Azure Pipelines. Except that on the latter, you can use `stage dependencies`. I kept looking for an alternative in GitHub Actions with no success so far.

Reply:
Can't we make two jobs: the first is the a job named `ci_skip`, and the second job is this job with a `needs: ci_skip` with a condition, like:

https://github.com/pgxn/docker-pgxn-tools/blob/main/.github/workflows/cicd.yml#L24-L25

Reply:
I'll try... But does it mean that the 'skip' job needs to fail for the rest to run?

Reply:
In that case, I think the condition might not be necessary